src/containers/CodesplainAppBar.jsx

```diff
@@ -185,23 +217,34 @@ export class CodesplainAppBar extends Component {
 CodesplainAppBar.propTypes = {
   hasUnsavedChanges: PropTypes.bool.isRequired,
   userSnippets: CustomPropTypes.snippets,
+  username: PropTypes.string.isRequired,
+  token: PropTypes.string,
+  avatarURL: PropTypes.string,
 };
 
 CodesplainAppBar.defaultProps = {
   userSnippets: {},
+  token: undefined,
```

Comment:
This should probably be an empty string, since it is a string type  (and will evaluate to false, so it won't break any of your code)